mat64/matrix.go

```diff
@@ -340,51 +340,13 @@ func Det(a Matrix) float64 {
 
 // Dot returns the sum of the element-wise products of the elements of a and b.
 // Dot panics if the matrix sizes are unequal.
```

Comment:
Could you also update the comment here and remove the "element" stutter in the line above?